src/main/scala/com/goyeau/kubernetes/client/KubernetesClient.scala

```diff
@@ -9,7 +9,7 @@ import com.goyeau.kubernetes.client.util.SslContexts
 
 import scala.concurrent.ExecutionContext
 
-case class KubernetesClient[F[_]: ConcurrentEffect](httpClient: Client[F], config: KubeConfig) {
+case class KubernetesClient[F[_]: ConcurrentEffect](httpClient: Client[F], config: KubeConfig)(implicit cs: ContextShift[IO]) {
```

Comment:
```suggestion
case class KubernetesClient[F[_]: ConcurrentEffect: ContextShift](httpClient: Client[F], config: KubeConfig) {
```
With this we are not forcing the IO monad

Reply:
No need anymore. Changed to `Async`